### ISSUE TITLE
Drag&drop attachment doesn't create valid or usable URL.

### ISSUE THREAD

**Issue by Pascalio** [Zettlr/Zettlr#1411]
## Description

On Linux (at least), attachments dragged-n-dropped from right-hand attachment bar don't open in default application, instead they open in the browser. This makes the feature sort of unusable because as soon as the absolute path contains characters forbidden in URL's, the browser won't cooperate. And anyway, it shouldn't respect the default application settings.
Clicking on the same file from the attachment bar opens it in the correct application. The bug only happens when you click a file link in the note itself.

## Reproducing

100% reproducible :1st_place_medal: 
1. have an attachment at a path that contains illegal URL characters (like whitespaces or non-English letters)
2. drag it from attachment sidebar onto your document to auto create the link.
3. ctrl-click the link: it opens the browser at that location, and it can't handle it.

## Expected behaviour

Clicking the link, fires up the programme associated to the file type.
**In other words**: the drag&drop in step 2. should create a _local file_ URL. If you manually prepend "file://" to the URL, clicking it will start the correct programme. This URL formatting should happen automatically, or something else telling the system that we're dealing with a local path.

## Platform
 - OS and version: Linux, Ubuntu 20.04
 - Zettlr Version: 1.7.5

**Comment by nathanlesage** [comment]
> (like whitespaces or non-English letters)

Neither whitespaces nor non-ASCII-letters are invalid within URLs, as every browser automatically converts them to valid escaped characters in the background.

The error was something different, but it's fixed now.
